lib/game.js

```diff
@@ -68,43 +76,47 @@ function drawGround () {
   ceiling.move(groundDirection)
 }
 
+function drawBird (direction) {
+  bird.draw(context)
+  bird.fly(direction)
+}
+
 function checkForCollision () {
   obstacles.forEach(function(obstacle) {
     if (bird.x < obstacle.x + obstacle.width &&
```

Comment:
could you maybe write a function for finding the corners of an obstacle/the bird? just to avoid doing all this adding and checking for corners manually